ansible_collections/arista/avd/roles/eos_designs/python_modules/mlag/avdstructuredconfig.py

```diff
@@ -133,11 +135,7 @@ def vlan_interfaces(self):
             "mtu": self._p2p_uplinks_mtu,
         }
         if not self._mlag_l3:
-            return strip_empties_from_dict(
-                {
-                    main_vlan_interface_name: main_vlan_interface,
-                }
-            )
+            return [strip_empties_from_dict({"name": main_vlan_interface_name, **main_vlan_interface})]
```

Comment:
I think you can just add `name` above instead of inserting it later.

Reply:
fixed